src/components/cards/FetchTweetCard.tsx

```diff
@@ -47,9 +51,17 @@ const FetchTweetCard = ({ disabled, onFetchTweet }: FetchTweetCardProps) => {
       const { text } = data;
       const { users, media, tweets } = includes;
 
+      let isAnyMedia = false;
+      let isAnyReferencedTweet = false;
+
+      if (tweets) {
+        isAnyReferencedTweet = tweets.some((tweet: BaseTweetProps) => tweet.hasOwnProperty("id"));
+      }
+      if (media && media.length) isAnyMedia = true;
```

Comment:
what if the image sent is more than 1?
I know that p4e takes only first image, but the text will still have more than 1 link right?

Reply:
I have checked this, and apparently only one link will appear if the tweet contains image(s). So number of image =/= number of link.

Tested on staging:
https://twitter.com/jogjamakanterus/status/1623640775845376000
![Screen Shot 2023-02-09 at 20 10 16](https://user-images.githubusercontent.com/14356826/217821844-2a8638d8-37a4-4e33-a8d3-82b29e89bf50.png)

With this, the filter works.